qemu/tests/virtio_fs_share_data.py

```diff
@@ -455,7 +455,24 @@ def viofs_svc_stop_start(action, cmd, expect_status):
                     session.cmd(cmd_make % fs_dest, io_timeout)
                     status, output = session.cmd_status_output(
                         cmd_pjdfstest % fs_dest, io_timeout)
-                    if status != 0:
+                    failed_test = output.split("-------------------")[1]\
+                                        .split("Files=")[0]
```

Comment:
How about using "Result:" instead of "Files="

Reply:
Hello @xiagao , I believe there is NOT needed to be changed.   
Both the line including "Files=" and the line including "Results:" are NOT concerned in this case. We only care the lines which  starts with "/mnt" in test report summary.

Reply:
Hi @nickzhq ,I just think 'Result' is better in case 'Files' is found in other lines.It depends on you, 'Files=' also works to me.

2022-09-06 04:23:27: Test Summary Report
2022-09-06 04:23:27: -------------------
2022-09-06 04:23:27: [31m/mnt/myfs/pjdfstest/tests/rename/21.t         (Wstat: 0 Tests: 16 Failed: 3)[0m
2022-09-06 04:23:27: [31m  Failed tests:  [0m[31m12-14[0m
2022-09-06 04:23:27: [31m/mnt/myfs/pjdfstest/tests/symlink/03.t        (Wstat: 0 Tests: 6 Failed: 2)[0m
2022-09-06 04:23:27: [31m  Failed tests:  [0m[31m1-2[0m
2022-09-06 04:23:27: Files=235, Tests=8813, 225 wallclock secs ( 3.21 usr  0.87 sys + 29.24 cusr 83.32 csys = 116.64 CPU)
2022-09-06 04:23:27: Result: FAIL
2022-09-06 04:23:27: [root@bootp-73-75-69 ~]#
2022-09-06 04:23:27: 1